docs_source_files/content/_index.en.md

```diff
@@ -33,21 +33,20 @@ import org.openqa.selenium.By;
 import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.support.ui.WebDriverWait;
-
 import static org.openqa.selenium.support.ui.ExpectedConditions.presenceOfElementLocated;
 
 public class HelloSelenium {
 
     public static void main(String[] args) {
-        WebDriver driver = new FirefoxDriver();
+        WebDriver driver = new ChromeDriver();
         WebDriverWait wait = new WebDriverWait(driver, 10);
         try {
             driver.get("https://google.com/ncr");
             driver.findElement(By.name("q")).sendKeys("cheese" + Keys.ENTER);
-            WebElement firstResult = wait.until(presenceOfElementLocated(By.cssSelector("h3>a")));
-            System.out.println(firstResult.getText());
+            WebElement firstResult = wait.until(presenceOfElementLocated(By.cssSelector("h3>div")));
```

Comment:
Would be nice to change the selector in all examples, right?

Reply:
> Would be nice to change the selector in all examples, right?

@diemol , updated in all language bindings in all index files